lighthouse-core/audits/seo/font-size.js

```diff
@@ -208,7 +208,6 @@ class FontSize extends Audit {
       analyzedFailingNodesData,
       analyzedFailingTextLength,
       failingTextLength,
-      visitedTextLength,
```

Comment:
I removed `visitedTextLength` because we can now easily visit every node (it's one protocol call).